liblangutil/SourceReferenceFormatter.cpp

```diff
@@ -210,5 +214,11 @@ void SourceReferenceFormatter::printErrorInformation(ErrorList const& _errors)
 
 void SourceReferenceFormatter::printErrorInformation(Error const& _error)
 {
-	printExceptionInformation(SourceReferenceExtractor::extract(m_charStreamProvider, _error));
+	SourceReferenceExtractor::Message message = SourceReferenceExtractor::extract(
+													m_charStreamProvider,
+													util::Exception(_error),
+													Error::errorSeverity(_error.type())
+												);
+	message.errorId = _error.errorId();
```

Comment:
Why did you remove `_typeOrSeverity` arg from the second overload of `extract()`? I mean, the function is useless if you have to bypass it and move the ID over manually. I think it should have worked. If it does not then in the end changing the order of args would be better than this.